### PR TITLE
add completionData to promise reject

### PR DESCRIPTION
**Pull request by matanwerbner**
To resolve https://dev.azure.com/ChargeAfter/ChargeAfter%202.0/_workitems/edit/23795

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.3.2",
+  "version": "1.3.3",
```
Comment (kostat) [inline package.json]: The version is automatically managed via release process

File: src/index.tsx [hunk 2]
```diff
@@ -157,8 +157,10 @@ const launchPaymentsUI = (
         error?: CallbackStatus,
       ) => {
         if (error) {
-          console.error(`payments error: ${JSON.stringify(error)}`);
-          reject(error);
+          console.info(
```
Comment (kostat) [inline src/index.tsx:160]: is there anything "interesting" in the `data` that worth printing to console?

**Review by kostat — CHANGES_REQUESTED** [review]

File: src/index.tsx (hunk 2 above)
Comment (matanwerbner) [reply src/index.tsx:160]: In my opinion no, we can remove this print as it contains internal implementation data

Comment (matanwerbner) [reply src/index.tsx:160]: @kostat Do you want to remove the console, or add to it?

**Review by kostat — APPROVED** [review]

**Pull request merged**